content/applications/hr/payroll/reporting.rst

```diff
@@ -2,68 +2,103 @@
+the previous time period or year (a :guilabel:`Comparison` option).
 
-.. image:: reporting/favorites.png
-   :align: center
-   :alt: Favorite options for reports.
+Click on a parameter to select it and activate it. The report is immediately updated with the new
+parameters.
+
+The updated report can be set as a favorite report, meaning the parameters are stored for quick
+access in the future. First, click :guilabel:`Save the current search` under the
+:guilabel:`Favorites` section, and two options and a :guilabel:`Save` button appear.
+
+If the current report is desired to be the default when the report is accessed, tick the box next to
+:guilabel:`Default filter`. If the current report should be accessible to everyone in the database,
+tick the box next to :guilabel:`Share`.
+
+Finally, click :guilabel:`Save`, which saves the currently configured report, and it now appears
+beneath :guilabel:`Favorites`.
+
+Default reports
+===============
 
 Payroll
 -------
 
-Click on :menuselection:`Payroll --> Reporting --> Payroll` to display the :guilabel:`Payroll
-Analysis` report.
+Click on :menuselection:`Payroll app --> Reporting --> Payroll` to display the :guilabel:`Payroll
+Analysis` report. This report shows all the payslips generated in the last 365 days.
 
 .. image:: reporting/payroll-report.png
    :align: center
    :alt: Report dashboard view.
 
-This shows all the payslips generated in the last 365 days. The report can display metrics for a
-variety of parameters. Click the :guilabel:`Measures` box to view a drop-down menu with the various
-options to display. The options available include: :guilabel:`# Payslip`, :guilabel:`Basic Wage`,
+The report can display metrics for a variety of parameters. Click the :guilabel:`Measures` box to
+view a drop-down menu with the various options to display.
+
+The default options available include: :guilabel:`# Payslip`, :guilabel:`Basic Wage`,
 :guilabel:`Basic Wage for Time Off`, :guilabel:`Days of Paid Time Off`, :guilabel:`Days of
 Unforeseen Absence`, :guilabel:`Days of Unpaid Time Off`, :guilabel:`Gross Wage`, :guilabel:`Net
 Wage`, :guilabel:`Number of Days`, :guilabel:`Number of Hours`, :guilabel:`Work Days`,
 :guilabel:`Work Hours`, and :guilabel:`Count`.
 
+:guilabel:`Net Wage` is the default metric for the :guilabel:`Payroll` report.
+
 .. image:: reporting/measures.png
    :align: center
    :alt: Report dashboard view.
 
 Line chart
 ~~~~~~~~~~
 
-A line chart is the default view for the *Payroll Analysis* report. If a different view was
-selected, click on the :guilabel:`Line Chart` icon (middle icon) in the menu bar to change the view
-back to a line chart. An option to display the line chart in ascending or descending order appears
-at the end of the options.
+A line chart is the default view for the *Payroll* report. If a different view is selected, click
+the :guilabel:`Line Chart` icon (middle icon) in the menu bar to change the view back to a line
```

Comment:
OK- I was worried for people who couldn't see the image, that the text would need to be thorough enough. I will remove them since they SHOULD be obvious by their image and icon.